### DISCUSSION TITLE
More Valid Wands & less limited by the Word Wand

### DISCUSSION THREAD

**Discussion by ScythiaThe1st** [littlesera/LSCG#528] [Ideas]
because I really wanna slap someone with a tablet and say, shhh sleepy sleepy now,
or that time I wanted some keys to do something other than look pretty and unlock stuff,
maybe have a Gavel called Necro Smasher from TF2 have solid functions like engineer's wrench to bap I mean uprade people with
and maybe I've finally come over here when I tried to make a catnip wand out of a pet toy

**Comment by littlesera** [comment]
Implemented in #546 
Add the "wand", "magic" or "enchanted" keyword to a crafted item name/description to allow it to cast spells
